### DISCUSSION TITLE
Server ports/protocol when using local machine behind home router as server component

### DISCUSSION THREAD

**Discussion by 7router7** [Ysurac/openmptcprouter#2554] [Q&A]
Hi all, 
I am performing some tests using a local vm behind my home router as server component.
Can you share the ports and protocols used at server side for a correct configuration? Is it possible to customize them from server and client side?
Thanks and regards.

luca

**Comment by Ysurac** [comment]
Ports used on the server side are available here: https://github.com/Ysurac/openmptcprouter/wiki/Install-or-update-the-VPS
Often is easier to open range 65000 to 65534

**Comment by 7router7** [comment]
your immediate answer is really appreciated.
Thanks and regards.
